resources/prometheus/prometheus-rules.yaml

```diff
@@ -454,6 +454,78 @@ spec:
             )
           record: central:sli:availability:extended_avg_over_time28d
 
+        # - Queries the 90th percentile of central's handled GRPC/HTTP API requests latencies over the last 10 minutes.
+        - expr: |
+            (histogram_quantile(0.9, sum  by(le, namespace, grpc_service, grpc_method) (rate(grpc_server_handling_seconds_bucket{container="central", grpc_method!~"ScanImageInternal|DeleteImages|EnrichLocalImageInternal|RunReport|ScanImage|TriggerExternalBackup|Ping"}[10m]))) > 0) < bool 0.1
+          record: central:grpc_server_handling_seconds:rate10m:p90
+        - expr: |
+            (histogram_quantile(0.9, sum  by(le, namespace, path) (rate(http_incoming_request_duration_histogram_seconds_bucket{container="central", code!~"5.*|4.*", path!~"/api/extensions/scannerdefinitions|/api/graphql|/sso/|/|/api/cli/download/"}[10m]))) > 0) > 0.1
+          record: central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90
+
+        # - Queries the current central API latency (GRPC and HTTP) SLI by calculating the ratio of successful
+        #   instances of central:xxx:rate10m:p90 over its total instances for a certain period.
+        # - Note that to get the current SLI with a variable PERIOD, simply run the following query where PERIOD is the desired period in
+        #   promql duration format. This query is useful for dynamically determining an SLI regardless of an SLO.
+        #
+        #   sum_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[PERIOD]) / count_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[PERIOD])
+        #
+        - expr: |
+            sum_over_time(central:grpc_server_handling_seconds:rate10m:p90[28d]) / count_over_time(central:grpc_server_handling_seconds:rate10m:p90[28d])
+          record: central:grpc_server_handling_seconds:rate10m:p90:sli
+        - expr: |
+            sum_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[28d]) / count_over_time(central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90[28d])
+          record: central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:sli
+
+        # - Queries the error rate or the ratio of the instances of central:xxx:rate10m:p90
+        #   were equal to 0 over the total instances of central:xxx:rate10m:p90 within a period.
+        - expr: |
+            1 - central:grpc_server_handling_seconds:rate10m:p90:sli
```

Comment:
Is there a reason you changed the order in the naming compared to existing metrics? E.g. `central:grpc_server_handling_seconds:rate10m:p90:sli` vs `central:sli:grpc_server_handling_seconds:rate10m:p90`.